content/en/docs/contributing/crds.md

```diff
@@ -20,11 +20,10 @@ This will also update the version conversion code if needed.
 
 ## Versions
 
-cert-manager at time of writing has 4 CRD versions in use.
+cert-manager at time of writing has 4 CRD versions in use- `v1alpha2`, `v1alpha3`, `v1beta1` and `v1`. `v1alpha2`, `v1alpha3` and `v1beta1` are getting deprecated in cert-manager `v1.4.0` and will be removed in `v1.6.0`.
```

Comment:
suggestion: would this look better as a list?

```suggestion
cert-manager currently has 4 CRD versions in use:

- `v1`
- `v1beta1` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
- `v1alpha3` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
- `v1alpha2` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
```

Reply:
Thanks a lot @SgtCoDFish 🙏🏼